packages/evm/src/opcodes/codes.ts

```diff
@@ -52,159 +52,173 @@ export type OpcodeList = Map<number, Opcode>
 type OpcodeEntry = { [key: number]: { name: string; isAsync: boolean; dynamicGas: boolean } }
 type OpcodeEntryFee = OpcodeEntry & { [key: number]: { fee: number } }
 
+// Default: sync and no dynamic gas
```

Comment:
Can you explain what all of this is doing?  I'm not anxious about the change per se but what is this achieving for the purposes of tree-shaking?

Reply:
This is not tree shaking, this is simply code compactification chopping of a few bytes. I would also argue here that this reads *better* than before than going through this endless list since it more prominently points out the deviations, e.g. `dynamicGasOp('MLOAD')`.

Will keep this one.